plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListenerConfig.java

```diff
@@ -105,4 +111,25 @@ public OpenLineageListenerConfig setNamespace(String namespace)
         this.namespace = Optional.ofNullable(namespace);
         return this;
     }
+
+    @NotNull
+    @NotEmpty
```

Comment:
Remove `@NotNull`. `@NotEmpty` is sufficient.

Reply:
Done